google-chart.html

```diff
@@ -367,7 +367,15 @@
       this._draw(this._chart, this._dataView);
     },
 
+    /**
+    * Renders the chart using the provided data.
+    * @param {Object|undefined} chart Internal Google Visualization chart object.
+    * @param {google.visualization.DataView|undefined} data  Internal data state
+    */
     _draw: function(chart, data) {
+      if(chart == undefined || data == undefined) {
```

Comment:
Best to check against `null` to cover both `null` and `undefined` cases.

Reply:
In Polymer 2, observers don't wait for all values to be defined anymore. So this is getting triggered before data has been properly constructed as a DataTable. I will make the changes to check against null.